tests/testutil/service.go

```diff
@@ -553,15 +553,16 @@ func (ts *TestService) GetKubeEndpointURLs() []string {
 }
 
 // GetKubeFirstClusterDomain returns first cluster domain
-func (ts *TestService) GetKubeFirstClusterDomain() string {
+func (ts *TestService) GetKubeFirstClusterDomain() (string, string) {
 	args := []string{"get", "clusterdomains",
-		"-o", `jsonpath="{.items[0].metadata.name}"`}
+		"-o", `jsonpath='{.items[0].metadata.name}{"\t"}{.items[0].spec.addresses[0].ip}'`}
 	clusterDomain, err := KubectlCmd(args)
 	if err != nil {
 		ts.T.Fatalf("Failed to get the first Cluster Domain:  %v", err.Error())
-		return ""
+		return "", ""
```

Comment:
Is this supposed to be double quoted ?

Reply:
I added a return value to the function, but actually I don't think that will ever get called due to the Fatal before it, so I can remove that return entirely